src/app/pages/task-calendar/cloudsync/cloudsync-form/cloudsync-form.component.ts

```diff
@@ -231,7 +231,7 @@ export class CloudsyncFormComponent implements OnInit {
     name: 'cloudsync_picker',
     placeholder: T('Schedule the Cloud Sync Task'),
     tooltip: T('Choose one of the convenient presets\
```

Comment:
To match the tooltips:
"Select a schedule preset or choose <i>Custom</i> to open the advanced scheduler."